### PR TITLE
Add information about dependency on version 2.26 and later of the desktop application

### PR DESCRIPTION
**Pull request by Oxalin**
The minimum version of the desktop application required to properly work with v3 of the extension was missing.

Add the information to the README.md file, into the notification tab and add a link to lead the user to where they should download the desktop application (official website).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "buttercup-browser-extension",
-  "version": "3.0.0",
+  "version": "3.1.1",
```
Comment (perry-mitchell) [inline package.json:3]: No need to change the version. This is handled during release.

File: source/shared/library/version.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
 // Do not edit this file - it is generated automatically at build time
 
-export const BUILD_DATE = "2024-03-27";
-export const VERSION = "3.0.0";
+export const BUILD_DATE = "2024-03-31";
+export const VERSION = "3.1.1";
```
Comment (perry-mitchell) [inline source/shared/library/version.ts:4]: This could be reverted.

**Review by perry-mitchell — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Oxalin) [reply package.json:3]: Will drop it. However, I just want to point out that the source code is still at 3.0.0, even though v3.1.0 is in the wild.

**Comment by Oxalin** [comment]
This should be good to go with the dropped commit.

**Review by perry-mitchell — APPROVED** [review]

**Pull request merged**